### PR TITLE
Add search aa1 to implement 1

### PR DESCRIPTION
**Pull request by pinterid**
**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [x] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [x] New feature (non-breaking change which adds functionality)
 - [ ] Documentation (non-breaking change which adds documentation)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- Currently there is no atom for SearchBar. #50 

**What is the new behavior (if this is a feature change)?**
- Now there is an atom for SearchBar. #50 

**Other information**:
- Ref: 🐍

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -520,6 +520,26 @@ class App extends React.Component {
     });
   };
 
+  /**
+   * Get all users
```
Comment (pinterid) [inline src/App.js:524]:
```suggestion
   * Get all users
   *
```

**Review by pinterid — COMMENTED** [review]
Looks good after a change.

**Review by pinterid — COMMENTED** [review]
Looks good.

File: src/App.js [hunk 2]
```diff
@@ -520,6 +520,27 @@ class App extends React.Component {
...
+   */
+  getAllPageUrls = () => {
+    return this.session.tasks.general.allPageUrls().then((res) => {
+      let urls = [];
+
+      res.data.pages &&
+        res.data.pages.forEach((page) => {
+          if (page.urlPath.includes("registration/")) {
+            let url = page.urlPath.split("/")[2];
+            urls.push(url);
```
Comment (schettn) [inline src/App.js]:
```suggestion
            let url = page.urlPath.split("/")[2];
            
            urls.push(url);
```

**Review by schettn — CHANGES_REQUESTED** [review]
I left a suggestion.

**Review by schettn — APPROVED** [review]
LGTM!

**Pull request merged**